src/main.rs

```diff
@@ -67,3 +64,51 @@ fn App() -> Element {
         }
     }
 }
+
+#[component]
+fn Registrations(selected_race_id: Signal<Option<u32>>) -> Element {
```

Comment:
Will this component make a sense when theres no selected race? I would render it only if race is selected